lib/OpenQA/Scheduler.pm

```diff
@@ -32,36 +33,12 @@ use constant SCHEDULE_TICK_MS => $ENV{OPENQA_SCHEDULER_SCHEDULE_TICK_MS} // 2000
 
 our $RUNNING;
 
-sub run {
-    my $self   = __PACKAGE__->new;
-    my $daemon = $self->setup;
-    local $RUNNING = 1;
-    $daemon->run;
-}
-
-sub setup {
-    my $self = shift;
-
-    my $setup = OpenQA::Setup->new(log_name => 'scheduler');
-    OpenQA::Setup::read_config($setup);
-    OpenQA::Setup::setup_log($setup);
-
-    log_debug("Scheduler started");
-    log_debug("\t Scheduler default interval(ms) : " . SCHEDULE_TICK_MS);
-    log_debug("\t Max job allocation: " . OpenQA::Scheduler::Model::Jobs::MAX_JOB_ALLOCATION());
-
-    # initial schedule
-    OpenQA::Scheduler::Model::Jobs->singleton->schedule;
-    Mojo::IOLoop->next_tick(sub { _reschedule() });
-
-    return Mojo::Server::Daemon->new(app => $self);
-}
-
 sub startup {
     my $self = shift;
 
+    $self->_setup if $RUNNING;
```

Comment:
i'd put at least a info message that the schedule started

Reply:
What do you mean? The removed `Scheduler started` debug message? How is that useful for you? The constants still have debug log messages in the `_setup` method.

Reply:
Worth mentioning that the other services (webui, livehandler, websockets) do not have a `$SERVICE started` log message. So i'm opting for consistency here with the removal.

Reply:
Oh, i think i get it now. Historically the scheduler has been a custom daemon that did not provide any feedback when started. Unlike the other services which were HTTP servers and told you their address. But that's now true for the scheduler too. It is actually quite verbose by default.
```
$ script/openqa-scheduler daemon                                 
[info] Listening at "http://127.0.0.1:9529"
Server available at http://127.0.0.1:9529
[info] Listening at "http://[::1]:9529"
Server available at http://[::1]:9529
[debug] [pid:23039] Scheduler default interval(ms): 1000
[debug] [pid:23039] Max job allocation: 80
[debug] [pid:23039] [rescheduling] Current tick is at 1000 ms. New tick will be in: 1000 ms
```

Reply:
@kraih Yep I saw them, but they will run on a per tick basis. What I meant was more that from the perspective of the service, I don't know if the service has started, until the tick happens.

>  Worth mentioning that the other services (webui, livehandler, websockets) do not have a $SERVICE started log message. So i'm opting for consistency here with the removal.

Sorta, you will get the Mojolicious::Commands messages (`Server available at http://$addr:$port`) per service, they do not end up in the log however... So I guess I'm fine with this :)

Reply:
The tick happens immediately, it's like a zero second timer. It is used here to change the context in which the reschedule function is called. The timing of it should not change in a noticeable way.

Reply:
I think the `Listening at` and `Server available at` messages are sufficient.

Reply:
> I think the `Listening at` and `Server available at` messages are sufficient.

Indeed